docs/tools/index.md

```diff
@@ -0,0 +1,71 @@
+# Development tools
+
+## CiviCRM-specific tools
```

Comment:
For reference, we continued this discussion here: https://chat.civicrm.org/civicrm/pl/cy1ktuknjidezr49uy3dg6drre